### ISSUE TITLE
imgproc.color should be deprecated

### ISSUE THREAD

**Issue by 9il** [libmir/dcv#43]
It can be replaced with single template which calls ndslice.algorithm and colour transformations from upcoming Phobos `color`  module.

Dub: http://code.dlang.org/packages/color
Git: https://github.com/TurkeyMan/color
Colour thread at forum: https://forum.dlang.org/thread/mailman.31.1473653679.2994.digitalmars-d@puremagic.com

In addition, color transformation should use 2D representation where color is packed in the last dimension. This will optimise iteration.

**Comment by ljubobratovicrelja** [comment]
Yes, this should definitely be done. This seems like a very powerful library, and dcv should definitely take advantage of it! I'll have to get familiar with it, and see how exactly we can blend it in dcv.

Thanks!

**Comment by 9il** [comment]
See also https://github.com/libmir/mir/issues/343

**Comment by 9il** [comment]
see also [forum thread](http://forum.dlang.org/post/johwawivehqpilmpryjn@forum.dlang.org)
